packages/cf-style-provider/package.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
Ok, but let's do in a different PR. There should be yarn.lock in every package. Update travis.yml. Test that it works for private packages as well (cf-ux).